crates/shared/src/arguments.rs

```diff
@@ -75,6 +75,11 @@ pub struct OrderQuotingArguments {
     #[clap(long, env, use_value_delimiter = true)]
     pub price_estimation_drivers: Vec<ExternalSolver>,
 
+    /// A list of external drivers used for native price estimation in the
+    /// following format: `<NAME>|<URL>,<NAME>|<URL>`
+    #[clap(long, env, use_value_delimiter = true)]
+    pub native_price_estimation_drivers: Vec<ExternalSolver>,
```

Comment:
We already have `native_price_estimators` as an argument. Can we not simply configure the drivers inline using this one instead of adding yet another argument?

Reply:
I thought about it initially and IMO this would not work so well for 2 reasons:
1. there is OneInchSpotPriceApi which is usually enabled but is not a price estimation driver so it would be kind of odd to configure in the same argument.
2. to save on expensive API requests the native price estimators get configured in stages. This would have to be done with the driver urls as well.

Of course both of these things can be done but I think this would probably not end up making our lives any easier when configuring stuff in the infra repo.
I believe to configure that reasonably well you would have to have a separate flag like `enableSpotPriceApi` and every native price estimator would need a `stage` argument.
Then when building the final `--native-price-estimators` argument you would have to group and sort all drivers by stages and also fit the OneInchSpotPriceApi in there. IMO especially the `stage` argument within each solver would make it very inconvenient to figure out how the exact native price estimator stages end up looking.

Ultimately we'd trade off complexity in the rust code with complexity in the ts code and I think since ts a comparatively shitty language it's harder to manage complexity there.

Reply:
Even today `NativePriceEstimator` is an enum which can be parsed from different string representations, so I think we could simply configure it as:

```
OneInchSpotPriceApi,baseline|http://driver/baseline;zeroex|http://driver/zeroex
```

and keep the `,` vs `;` for specifying stages. This way you would avoid creating an invalid configuration where `native_price_estimators` is referencing a driver that doesn't exist.

Reply:
Yes, I think this is a great improvement!